src/ProxyManager/Generator/MethodGenerator.php

```diff
@@ -21,10 +23,83 @@ class MethodGenerator extends ZendMethodGenerator
     public static function fromReflection(MethodReflection $reflectionMethod) : self
     {
         /* @var $method self */
-        $method = parent::fromReflection($reflectionMethod);
+        $method = self::fromReflectionWithoutDocBlock($reflectionMethod);
 
         $method->setInterface(false);
 
         return $method;
     }
+
+    /**
+     * @see \Zend\Code\Generator\MethodGenerator::fromReflection
+     */
+    private static function fromReflectionWithoutDocBlock(MethodReflection $reflectionMethod)
```

Comment:
Isn't it easier to just introduce a `parent::` call and set the docblock generator in here?

Also, please don't squash - comments were already being hidden because of the squash...

Reply:
I'm not sure what you mean. I could move the whole `fromReflectionWithoutDocBlock` to `fromReflection` directly. But then it's not that abvious what has changed.

Reply:
@Tobion I'm mainly concerned with extracting logic from `zend-code`. A `removeDocblock()` method on the generator would be best, and it can probably easily be implemented in `zend-code` directly. That's what I'm suggesting, basically: we should use the normal facilities, then remove the docblocks later.

Reply:
If you mean to call the parent and then `$method->setDocBlock(null)`, it would still execute `\Zend\Code\Generator\DocBlockGenerator::fromReflection` for nothing.

Reply:
>  it would still execute `\Zend\Code\Generator\DocBlockGenerator::fromReflection` for nothing.

Not a problem. This is not performance sensitive code.